stdlib/asm/mem.masm

```diff
@@ -84,8 +84,8 @@ end
 #! Input: [num_words, write_ptr, ...]
 #! Output: [HASH, write_ptr', ...]
 #! Cycles:
-#!  even num_words: 48 + 9 * num_words / 2
-#!  odd num_words: 65 + 9 * round_down(num_words / 2)
+#!  even num_words: 50 + 9 * num_words / 2
+#!  odd num_words: 67 + 9 * round_down(num_words / 2)
 export.pipe_words_to_memory.0
```

Comment:
I'm wondering if we should change the semantics of this procedure to make it consistent with the `pipe_double_words_to_memory` procedure above. Specifically, this procedure could have the following interface:
```
#! Input:  [num_words, write_ptr, ...]
#! Output: [C, B, A, write_ptr', ...]
```
Then, we could squeeze the digest out in `pipe_preimage_to_memory` rather than here.

Reply:
Done